int16 mark_status
int16 mark_id
---
int16 mark_status
int16 mark_id
float64 linear_x
float64 linear_y
float64 angular_z
